Assets/Text/cqui_text_settings_de.xml

```diff
@@ -238,6 +238,24 @@
     <Row Tag="LOC_CQUI_SHOW_WAR_ICON_IN_CITYSTATE_BANNER_TOOLTIP" Language="de_DE">
         <Text>Wenn Sie sich im Krieg mit einem Stadtstaat befinden, zeigen Sie die Kriegsikone im Stadtbanner dieses Stadtstaates</Text>
     </Row>
+    <Row Tag="LOC_CQUI_LABELS_ON_SUZERAIN_ICON" Language="de_DE">
+        <Text>Suzerain Envoy Labels anzeigen</Text>
+    </Row>
+    <Row Tag="LOC_CQUI_LABELS_ON_SUZERAIN_ICON_TOOLTIP" Language="de_DE">
+        <Text>Zeigen Suzerain Gesandten Etiketten in CityState Banner[NEWLINE]Wenn Sie Suzerain sind, zeigt die Zahl oben rechts des Symbols gesandte Gesandte an.[NEWLINE]Wenn Sie nicht Suzerain sind, zeigen Ihre Gesandten oben links, Suzerain Gesandte oben rechts.</Text>
+    </Row>
+    <Row Tag="LOC_CQUI_SHOW_SUZERAIN_CIV_ICON" Language="de_DE">
+        <Text>Show Civilization Icon</Text>
+    </Row>
+    <Row Tag="LOC_CQUI_SHOW_SUZERAIN_DISABLED" Language="de_DE">
+        <Text>Suzerain nicht anzeigen</Text>
+    </Row>
+    <Row Tag="LOC_CQUI_SHOW_SUZERAIN_IN_CITYSTATE_BANNER_TOOLTIP" Language="de_DE">
```

Comment:
remove the instance of this tooltip above (line 232)  (this text here is the updated one)